### PR TITLE
Fix Stripe config definition

### PR DESCRIPTION
**Pull request by slavcodev**
The returnUrl is not a credential, it is a setting

### CODE REVIEW

**Review by vova07 — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

**Review by vova07 — APPROVED** [review]
It will require F-E adjustments.

**Review by iamdevice — APPROVED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Stripe.yaml [hunk 1]
```diff
@@ -22,12 +18,16 @@ allOf:
...
-          - redirectUrl
       settings:
         type: object
         description: Stripe settings object
+        required:
+          - redirectUrl
         properties:
+          redirectUrl:
+            type: string
+            description: The URL where the user will be redirected after authorizing the account on Stripe.
```
Comment (adamaltman) [inline openapi/components/schemas/GatewayAccountConfig/Stripe.yaml:29]: Why is this a setting? Why isn't this following the same redirectUrl behavior we have everywhere?

Comment (slavcodev) [reply openapi/components/schemas/GatewayAccountConfig/Stripe.yaml:29]: This how it works on backend. We store this in settings. I believe changing the API should be separate project.

Comment (adamaltman) [reply openapi/components/schemas/GatewayAccountConfig/Stripe.yaml:29]: It seems this should be read only then. Stripe only allows our preconfigured URLs for oauth2.

Comment (slavcodev) [reply openapi/components/schemas/GatewayAccountConfig/Stripe.yaml:29]: No, it doesn't. [Stripe accept redirect URL dinamically](https://stripe.com/docs/connect/oauth-reference#redirect-uri)

**Pull request merged**